recipes/zxing-cpp/all/conanfile.py

```diff
@@ -113,6 +113,7 @@ def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "ZXing")
         self.cpp_info.set_property("cmake_target_name", "ZXing::ZXing")
         self.cpp_info.set_property("pkg_config_name", "zxing")
+        self.cpp_info.includedirs = [os.path.join("include", "ZXing")]
```

Comment:
I also noticed that the `zxing-cpp` recipe is incorrect in its exported include dirs compared to the official exports, but I really don't like the library's choice of leaving out the prefix from its include paths. It adds some pretty bad header names like `Error.h`, `Result.h`, etc to the list of visible includes.
For that reason, I would prefer to use `self.cpp_info.includedirs.append(os.path.join("include", "ZXing"))` here and leave it out of this PR entirely.

Reply:
I'll revert this and open an PR only for ZXing at conan-centre. Append makes more sense indeed.